bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_widthx1p45_NNPDF31_NNLO_dilepton.input

```diff
@@ -0,0 +1,80 @@
+! TTbar production parameters
+!randomseed 352345 ! uncomment to set the random seed to a value of your choice.
+                   ! It generates the call RM48IN(352345,0,0) (see the RM48 manual).
+                   ! THIS MAY ONLY AFFECTS THE GENERATION OF POWHEG EVENTS!
+                   ! If POWHEG is interfaced to a shower MC, refer to the shower MC
+                   ! documentation to set its seed.
+
+
+
+numevts NEVENTS
+iseed SEED
+ih1   1        ! hadron 1
+ih2   1        ! hadron 2
+
+
+! To be set only if using LHA pdfs
+lhans1 306000
+lhans2 306000
+! NNPDF31_nnlo_as_0118
+ebeam1 6500    ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+qmass  172.5     ! mass of heavy quark in GeV
+facscfact 1    ! factorization scale factor: mufact=muref*facscfact 
+renscfact 1    ! renormalization scale factor: muren=muref*renscfact 
+
+hdamp 237.8775
+
+topdecaymode 22200   ! an integer of 5 digits that are either 0, or 2, representing in 
+                     ! the order the maximum number of the following particles(antiparticles)
+                     ! in the final state: e  mu tau up charm
+                     ! For example
+                     ! 22222    All decays (up to 2 units of everything)
+                     ! 20000    both top go into b e nu (with the appropriate signs)
+                     ! 10011    one top goes into electron (or positron), the other into (any) hadrons,
+                     !          or one top goes into charm, the other into up
+                     ! 00022    Fully hadronic
+                     ! 00002    Fully hadronic with two charms
+                     ! 00011    Fully hadronic with a single charm
+                     ! 00012    Fully hadronic with at least one charm
+
+!semileptonic 1      ! uncomment if you want to filter out only semileptonic events. For example,
+                     ! with topdecaymode 10011 and semileptonic 1 you get only events with one top going
+                     ! to an electron or positron, and the other into any hadron.
+
+! Parameters for the generation of spin correlations in t tbar decays
+tdec/wmass 80.4  ! W mass for top decay
+tdec/wwidth 2.141
+tdec/bmass 4.8
+tdec/twidth  1.5065 ! 1.45  x 1.31 central width
```

Comment:
this line is still buggy. shouldn't it be 1.8995 ?

Reply:
Ok I can change that, thanks. What does the 1.8995 represent here? I originally copied this card and changed the width from 1.15 to 1.45: https://github.com/jfernan2/genproductions/blob/dafd0072238753e9b78a783da1e144ef84b7e2df/bin/Powheg/production/2017/13TeV/TT_hvq/TT_hdamp_widthx1p15_NNPDF31_NNLO_dilepton.input

Reply:
The top width at 172.5 GeV is 1.31 (called central width above). Now, the 1.45 that you indicate above is the relative width, i.e. 1.31 * 1.45 (45% increase) so 1.8995. @wajidalikhan and @GiuliaNegro please confirm that this is the intention and the setting should be changed according to my proposal.

Reply:
@agrohsje  yes, it has to be relative width as you mentioned.Thanks a lot for having a critical look. Really appreciated.